theme/sass/components/_blog-sidebar-component.scss

```diff
@@ -0,0 +1,31 @@
+.blog-sidebar-component {
+
+  @include breakpoint($md) {
+    width: 40%
+    float: right;
```

Comment:
This should be floating to the left.

Reply:
No, it should be right.

Reply:
They both work the same way so thats fine.